hieradata/class/jenkins.yaml

```diff
@@ -29,6 +29,11 @@ govuk_jenkins::config:
       --httpPort=$HTTP_PORT
       --ajp13Port=-1
       --sessionTimeout=1440
+  JAVA_ARGS:
+    value: >-
+      -Djava.awt.headless=true
```

Comment:
These are provided [by the upstream repo](https://github.com/jenkinsci/puppet-jenkins/blob/23df7764d58aed80b573621b102f82a702217a7a/manifests/params.pp#L28), so not sure if there is a good way to merge them together.